### PR TITLE
Nc news 6

### PR DESCRIPTION
**Pull request by YoosufAhmad**
created a response for when a user requests comments for a particular article using article id.

### CODE REVIEW

File: models/topics.model.js [hunk 1]
```diff
@@ -24,3 +24,13 @@ exports.selectAllArticles = () => {
...
 }
+
+exports.selectComments = (article_id) => {
+  const query = 'SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at desc;'
+  return db.query(query, [article_id]).then(({rows}) => {
+    if (!rows.length) {
+      return Promise.reject({ status: 404, msg: "Not found" });
+    }
+    return rows
+  })
```
Comment (Keb84) [inline models/topics.model.js:32]: We should be checking why we are getting an empty array here 👀 is it because the article exists but has no comments or is it because the article doesn't exist? You be will seeing the same result of an empty array for two very different reasons. Think about how you could check the articles table first to see if the ID exists 🤔 would you want to see an error first and if not then you could query the comments table 💭

File: __tests__/models.test.js [hunk 2]
```diff
@@ -129,3 +129,40 @@ describe.only('GET/api/articles', () => {
...
+      
+      comments.forEach(comment => {
+        expect(comment).toHaveProperty('votes', expect.any(Number))
+        expect(comment).toHaveProperty('body', expect.any(String))
+        expect(comment).toHaveProperty('author', expect.any(String))
+        expect(comment).toHaveProperty('comment_id', expect.any(Number))
+        expect(comment).toHaveProperty('article_id', expect.any(Number))
+        expect(comment).toHaveProperty('created_at', expect.any(String))
+      })
+    })
```
Comment (Keb84) [inline __tests__/models.test.js:150]: Another status 200 test would be nice here 👀 you could check if the article ID exists but the comments array is empty 🤸

**Review by Keb84 — CHANGES_REQUESTED** [review]
Nice work with this ticket 🤸 please make the suggested changes and resubmit 🎸
